src/repositories/compare.jl

```diff
@@ -0,0 +1,28 @@
+###################
+# Comparison Type #
+###################
+
+@ghdef mutable struct Comparison
+    url::Union{URIs.URI, Nothing}
+    html_url::Union{URIs.URI, Nothing}
+    permalink_url::Union{URIs.URI, Nothing}
+    diff_url::Union{URIs.URI, Nothing}
+    patch_url::Union{URIs.URI, Nothing}
+    base_commit::Union{Commit, Nothing}
```

Comment:
I think this is deprecated, if that matters

Reply:
I don't think it matters. We're just encoding the response here, so I guess it doesn't hurt if that includes deprecated fields (where did you find this?). They will just be set to `nothing` if the API stops returning them.